packages/server/src/service/static/views/WestVirginia.md

```diff
@@ -0,0 +1,12 @@
+{% extends "Base.md" %}
+{% from "Image.md" import image %}
+
+{% set guidance = '[ In accordance with guidelines from the office of the West Virginia Secretary Of State Mac Warner](https://sos.wv.gov/elections/Pages/AbsenteeVotingInformation.aspx)' %}
```

Comment:
nit: dunno if it even effects anything, but trailing space within brackets

Reply:
Not sure where you want a trailing whitespace?

Reply:
sorry, I meant extraneous leading whitespace

Reply:
Ah got it - resolved.